dockerFiles/marklogic-deps-centos:base

```diff
@@ -13,7 +13,10 @@ LABEL "com.marklogic.maintainer"="docker@marklogic.com"
 
 RUN echo "NETWORKING=yes" > /etc/sysconfig/network \
     && yum -y update \
-    && yum -y install gdb.x86_64 glibc.i686 initscripts redhat-lsb-core.x86_64 tzdata ntp \
+    && yum -y install gdb.x86_64 glibc.i686 initscripts redhat-lsb-core.x86_64 tzdata ntp epel-release yum-utils xz-0:5.2.2-2.el7_9.x86_64 \
+    && rpm -Uvh http://www.city-fan.org/ftp/contrib/yum-repo/rhel7/x86_64/city-fan.org-release-2-2.rhel7.noarch.rpm \
```

Comment:
We probably can't rely on this repository for a curl binary. See if we can find a better place to pull curl.